.claude/skills/new-garden-page/SKILL.md

```
@@ -0,0 +1,223 @@
+---
+```
+
+#### Page Name
+
+- Lowercase, hyphenated: `cold-brew`, `pile-of-shame`, `terrance-gore`
+- Short and descriptive
+- Check for collisions: `ls content/garden/<topic>/` to ensure no duplicate
+
+#### File Path
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)